### ISSUE TITLE
Calendar preventing clock in statusbar from Updating

### ISSUE THREAD

**Issue by Shadow-of-eagle** [FossifyOrg/Calendar#106]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Calendar/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Calendar/issues) or [closed](https://github.com/FossifyOrg/Calendar/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Calendar/discussions) or [closed](https://github.com/FossifyOrg/Calendar/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [x] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Calendar/CONTRIBUTING.md).

### Affected app version

1.0.2

### Affected Android/Custom ROM version

Android 8.1.0

### Affected device model

Sony Xperia L3

### How did you install the app?

Google Play Store

### Steps to reproduce the bug

I am not 100% sure whether the App caused the Problem, but there are quite some indications. More under "Additional information". 
To reproduce the app has just to be installed. I have not found out in what exactly causes this to happen.

### Expected behavior

Have no effect on the clock.

### Actual behavior

Sometimes, when the app is installed, the clock in the statusbar stops updating. All other clocks still update (Widget, Clock-app). Changing the Time format settings (and some other things) updates it once.

### Screenshots/Screen recordings

<img alt="Time_wrong" src="https://github.com/FossifyOrg/Calendar/assets/128985883/864a39e8-a8d5-4812-834a-8a692ab12929" width=200 />
<img alt="Time_wrong_2" src="https://github.com/FossifyOrg/Calendar/assets/128985883/5398ce0a-2839-4224-be35-123c758cedf6" width=200 />

### Additional information

Like already mentioned I am not 100% sure whether this is caused by the calendar, but the following speaks for it:
1. It started the day after I installed the apps (Calendar, Phone, Filemanager). I have not made any other changes to my device at that time. This happens quite often, so if it was caused by anything I did a week ago I would have noticed it earlier.
2. I uninstalled all of the Fossify Apps except the calendar, because I had already added some events, and it kept happening.
3. I uninstalled the Calendar and it did not happen again.
4. I reinstalled the Apps (about 12 hours later, maybe that was a bit to early to be sure) and it happened again.

Maybe someone else has the same problem and can confirm, that it is caused by the Calendar and help getting further information about how to reproduce it.

I apologize for any language errors and if I posted it at the wrong place, I was not sure where the best place would be.

**Comment by Aga-C** [comment]
It looks like some strange performance issue on your phone. Hard to tell what exactly can cause it.

If somebody else has a similar problem, please comment here, along with your device model and OS version.

**Comment by Shadow-of-eagle** [comment]
I am very sure, that this issue is caused by the calender, because since the last post I uninstalled the calendar and kept the other apps and it did not happen again. 
My brother has the same model and android version and installed the calendar too, but did not have the problem until now, so it is probably a problem of my specific device.
I used (and still use) Simple calendar Pro 6.23.1 without issues.
